pydra/engine/core.py

```diff
@@ -563,6 +570,7 @@ def add(self, task):
             else:
                 task.state = state.State(task.name, other_states=other_states)
         self.node_names.append(task.name)
+        self.graph_sorted
```

Comment:
perhaps the semantics of this can be improved. this looks weird in python. i would turn this into a function with a better name.